content/F2016/F16-event-teaparty.md

```diff
@@ -0,0 +1,21 @@
+title: Beginning of Term Tea Party
+Date: 2016-9-13
+Category: Events
+Tags: social, volunteer, big csters
+Slug: tea-party-f16
+Author: Anna Lorimer
+Summary: Come meet fellow women in CS over tea and snacks!
+
+Welcome back! You're cordially invited to our Tea Party and Volunteer meeting.
+Come out to socialize with some WiCS members, meet the new committee, and learn
+more about our volunteer opportunities!
+
+Note: Although our volunteer meeting is open to everyone, our Tea Party is
+a Big CSters event is and is thus [women-only]({filename}/pages/faq.md).
+
+## Event Details ##
+
++ **Who:** [Women and Non-binary individuals]({filename}/pages/faq.md)
```

Comment:
yeah that makes sense. I always so the capital letter as a beginning of sentence thing instead of women being a proper noun. 

I think if we capitalize non-binary we would want to capitalize Binary too?

Also - not urgent or anything but I wonder if we should add some guidelines around the standards for punctuation etc for this section since we often talk about consistency but don't actually have the 'chosen consistent' thing anywhere... just thinking ^_^